Finmer.Game/Models/GameUI.cs

```diff
@@ -150,6 +151,10 @@ public void Reset()
             IsScriptConsoleOpened = false;
             ControlsEnabled = false;
             InventoryEnabled = false;
+            //m_CombatStateViewModel = null;
+            s_Inst = new GameUI();
```

Comment:
Either approach is fine by me, but if you're going with `s_Inst = new GameUI` (which should be fine), then all the other cleanup in the function is unnecessary and can be removed (since it's all being discarded anyway); this is the only line of code that needs to be here still.